Source/Editor/Surface/Archetypes/Parameters.cs

```diff
@@ -937,13 +952,25 @@ public void OnParamDeleted(SurfaceParameter param)
             {
                 // Deselect if that parameter is selected
                 if ((Guid)Values[0] == param.ID)
-                {
                     _combobox.SelectedIndex = -1;
-                }
 
                 UpdateCombo();
             }
 
+            /// <inheritdoc />
+            public bool IsParamreferenced(SurfaceParameter param)
+            {
+                for (int i = 0; i < Surface.Nodes.Count; i++)
```

Comment:
This loop can be removed and whole function body can be:
```cs
return (Guid)Values[0] == param.ID;
```

Reply:
Added in https://github.com/FlaxEngine/FlaxEngine/pull/3010/commits/38d1e25604cc260fb2f60227a0bae90b43cc30ca